core/com.b2international.snowowl.core.rest.tests/src/com/b2international/snowowl/core/rest/codesystem/CodeSystemApiTest.java

```diff
@@ -423,4 +435,20 @@ public void codesystem23_DeleteVersioned() throws Exception {
 				.isDeleted()).isTrue();
 	}
 	
+	@AfterClass
+	public static void cleanUp() {
+		ResourceRequests
```

Comment:
Not sure if this is a great idea, at least the global SNOMED CT CodeSystem should not be deleted, otherwise tests will fail.

Reply:
Done